runtime/fundamentals/web_standards.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Web Standards and Node APIs"
+---
+
+Deno aims to bridge the gap between the browser and the backend with a set of
+web standard APIs that are familiar to web developers. These APIs are designed
+to be compatible with the web platform, making it easier to write code that can
+run in both the browser and Deno.
+
+Existing Node.js ecosystem compatibility is provided through a set of Node.js
+compatible APIs and Deno namespace APIs. These APIs are designed to be
+compatible with Node.js, making it easier to port existing Node.js code to Deno.
```

Comment:
The phrasing seems to suggest that the `Deno` namespace APIs help with node compatibility. Maybe something like:

```suggestion
Existing Node.js ecosystem compatibility is provided through a built-in compatibility layer that allows you to use any npm package directly in Deno.
```